compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/desugar/QueryDesugar.java

```diff
@@ -147,7 +147,11 @@ BLangStatementExpression desugar(BLangQueryExpr queryExpr, SymbolEnv env) {
         if (queryExpr.isStream) {
             streamStmtExpr = ASTBuilderUtil.createStatementExpression(queryBlock, streamRef);
             streamStmtExpr.type = streamRef.type;
-        } else {
+        }
+//        else if (queryExpr.isTable) {
+//            //TO DO
+//        }
```

Comment:
I didn't complete this part. should a method be implemented for table functionality too like the 'buildStream' method?

Reply:
No need for such a method, as this will be part of the existing pipeline. So the idea is, regardless of whether it's a stream, an array, a table, or an action, it will internally create a stream. That's the purpose of `buildStream` method. And, I'll add the method to create the table from stream later since it's yet to be finalized.